x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/action/ReloadAnalyzersResponse.java

```diff
@@ -99,7 +102,36 @@ public static ReloadAnalyzersResponse fromXContent(XContentParser parser) {
         return PARSER.apply(parser, null);
     }
 
-    public static class ReloadDetails {
+    @Override
+    public void readFrom(StreamInput in) throws IOException {
+        super.readFrom(in);
+        this.reloadDetails.putAll(in.readMap(StreamInput::readString, ReloadDetails::new));
```

Comment:
This should be done the ctor that takes StreamInput.  The readFrom should throw UOE.